README.md

```diff
@@ -1,6 +1,8 @@
 # Hedera Smart Contracts
 Reference library for Smart Contracts utilized by the Hedera network with supporting files and examples.
 
+Please checkout `v0.2.0` if you want to use the interface currently implemented in the Mainnet.
```

Comment:
I'm not sure this is the case or comment to address it.

The current version in main is deployed to Hedera Mainnet.
Is there an issue you're noting that you believe v0.2.0 this version doesn't match.
Notably there are changes since v0.2.0, these will be capture in a v0.3.0 release